Assets/MRTK/Core/Interfaces/InputSystem/IMixedRealityGazeProviderWithOverride.cs

```diff
@@ -0,0 +1,23 @@
+﻿// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+using UnityEngine;
+
+namespace Microsoft.MixedReality.Toolkit.Input
+{
+    /// <summary>
+    /// Adds ability to override head gaze on a gaze provider.
+    /// </summary>
+    public interface IMixedRealityGazeProviderWithOverride
```

Comment:
nit: maybe IMixedRealityGazeProviderHeadOverride?

Reply:
Yeah, I think I like yours better! Changed this a couple times and was never fully happy with it. Thanks!

Reply:
Renamed